pkg/operator/ceph/client/controller.go

```diff
@@ -234,11 +235,19 @@ func (r *ReconcileCephClient) createOrUpdateClient(cephClient *cephv1.CephClient
 	if err != nil {
 		key, err = ceph.AuthGetOrCreateKey(r.context, r.clusterInfo, clientEntity, caps)
 		if err != nil {
+			if strings.Contains(err.Error(), opcontroller.UninitializedCephConfigError) {
+				logger.Info("skipping reconcile since operator is still initializing")
+				return nil
```

Comment:
Should we not still return an error so the reconcile is re-queued? Or where does the re-queue happen for the ceph client controller?

Reply:
Good point.